app/assets/stylesheets/pregnancies.scss

```diff
@@ -10,6 +10,10 @@ $font-size: 17px;
   padding: 15px 0;
 }
 
+.voicemailOk{
```

Comment:
I think there's a bootstrap class that basically does this same thing? `text-center` iirc -- probably take this out to avoid redundancy?